src/pages/ResearcherProfile.js

```diff
@@ -406,17 +377,17 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     }
   };
 
-  updateResearcher(profile) {
+  updateResearcher = async (profile) => {
     const profileClone = this.cloneProfile(profile);
     Researcher.updateProperties(this.state.currentUser.dacUserId, true, profileClone).then(()=> {
```

Comment:
The nested ```.then()``` chains can be cleaned up with await statements like in ```saveProperties```

Reply:
Good point, will address